fixtures/link.fixture.js

```diff
@@ -0,0 +1,35 @@
+import Link from "@khanacademy/wonder-blocks-link";
+
+import {describe, expect, test, render, simulate, sleep} from "./helpers.js";
+
+function randomLink() {
+    return "#" + String(Math.random()).slice(2);
+}
+
+describe("Link", () => {
+    test("primary", async () => {
+        const element = await render(
+            <Link href={randomLink()}>Hello, world!</Link>,
+        );
+        await expect(element).toMatchScreenshot();
+        await simulate({type: "mousemove", clientX: 50, clientY: 20});
+        await expect(element).toMatchScreenshot();
+        await simulate({type: "click"});
+        await sleep(100);
```

Comment:
This is necessary b/c the browser doesn't update the style of visited links until the URL changes which takes some time.  We should probably wait for the URL change instead of hardcoding a set duration.